djangoplicity/events/templates/upcoming_options.html

```diff
@@ -8,9 +8,9 @@
 
     {% for key, value in choices %}
         {% if key == upcoming %}
-            <li><strong><a href="?upcoming={{ key }}{{ current_params }}{% if extra_params %}&{{ extra_params }}{% endif %}" style="color: black;">{{ value }}</a></strong></li>
+            <li><strong><a href="/public/events/?upcoming={{ key }}{{ current_params }}{% if extra_params %}&{{ extra_params }}{% endif %}" style="color: black;">{{ value }}</a></strong></li>
```

Comment:
Hi @JuanParrado19 

Keep in mind Djangoplicity is a project used by multiple organizations, therefore the URLs might vary, and also it's a bad practice to hardcode URLs. I believe keeping this as a relative URL is better than this change.